samples/apps/copilot-chat-app/webapi/PlannerSkills/Translate/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "schema": 1,
+  "type": "completion",
+  "description": "Translate the input into a language of your choice",
+  "completion": {
+    "max_tokens": 256,
+    "temperature": 0.7,
+    "top_p": 0.0,
+    "presence_penalty": 0.0,
+    "frequency_penalty": 0.0,
+    "stop_sequences": [
+      "[done]"
+    ]
```

Comment:
nit: consider adding input definitions.

Reply:
Keeping this as a copy/paste from the existing skills in the repo for now.